ga4gh/server/frontend.py

```diff
@@ -55,7 +55,7 @@
     'SECRET_KEY': "key",
     'TESTING': False,
     'DEBUG': False,
-    'OIDC_CLIENT_SECRETS': 'client_secrets.json',
+    'OIDC_CLIENT_SECRETS': '/srv/ga4gh/server/client_secrets.json',
```

Comment:
So this hardcoded path is going to (for instance) break Travis-CI tests.  Is there a way we can avoid this?

Reply:
This is a fix purely for the Docker implementation.  I'm thinking we should simply put this in a configuration file.

Reply:
A new pull request is in progress that will refactor the old authentication code in frontend.py and serverconfig.py. It is being used in the current deployer script and fixes the code syntax so that it passes the flake8 linter tests. I am still cleaning up the organization of the code, but I will also make the serverconfig.py script read in a separate configuration file in the same directory and hopefully separate the authentication code in frontend.py from the server configuration.